CMakeLists.txt

```diff
@@ -112,9 +113,15 @@ include_directories(${PROTOBUF_INCLUDE_DIRS})
 include_directories(${LIBGMP_INCLUDE_DIR})
```

Comment:
Should move this and the finding of the GMP package (line 110) under the ENABLE_GMP option.